tests/feature/AddressTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Tychovbh\Tests\Mvc\feature;
+
+use Illuminate\Support\Arr;
+use Tychovbh\Mvc\Address;
+use Tychovbh\Mvc\Http\Resources\AddressResource;
+use Tychovbh\Mvc\Service\AddressLookup\PdokService;
+use Tychovbh\Tests\Mvc\TestCase;
+
+class AddressTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function itCanIndex()
+    {
+        $address = factory(Address::class, 2)->create();
+        $this->index('addresses.index', AddressResource::collection($address));
+    }
+
+    /**
+     * @test
+     */
+    public function itCanShow()
+    {
+        $address = factory(Address::class)->create();
+        $this->show('addresses.show', AddressResource::make($address));
+    }
+
+    /**
+     * @test
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function itCanStore()
+    {
+        $address = factory(Address::class)->make(['zipcode' => '2352 CZ', 'house_number' => '38', 'country' => 'Nederland']);
+
+        $finalAddress = PdokService::search($address->zipcode, $address->house_number, $address->country);
```

Comment:
finalAddress should be used in AddressResource::make method. $address should be sent to the request params. This is now switched, so you are sending a complete address to the request, instead only zipcode and house_number should be sent.